src/ipc/ipc4/helper.c

```diff
@@ -751,29 +886,3 @@ struct comp_dev *ipc4_get_comp_dev(uint32_t comp_id)
 
 	return icd->cd;
 }
-
-int ipc4_add_comp_dev(struct comp_dev *dev)
```

Comment:
why is this removed? doesnt seem related to the change in this patch or I am not able to connect immediately

Reply:
That function isn't called anywhere outside the file where it's defined, so I made it static